.maestro/autofill/smoke.yaml

```diff
@@ -1,8 +1,9 @@
 appId: com.duckduckgo.mobile.android
-name: Autofill Smoke Tests
+name: "SmokeTest: Autofill screen is reachable from overflow menu"
+tags:
+    - releaseTest
 ---
 - launchApp:
     clearState: true
 - runFlow: ../shared/onboarding.yaml
-- runFlow: 1_autofill_shown_in_overflow.yaml
-- runFlow: 2_autofill_reach_creds_management.yaml
\ No newline at end of file
+- runFlow: 1_autofill_shown_in_overflow.yaml
```

Comment:
you’ve removed the execution of `2_autofill_reach_creds_management.yam`. just checking it’s intentional

Reply:
Yeah, intentional. that one is dependent on external device configuration being a certain way and therefore isn't a good release/smoke testing step.